content/200-concepts/050-overview/500-under-the-hood.mdx

```diff
@@ -75,22 +77,31 @@ Run the following command to set the environment variable globally (in this exam
 
    <tab>
 
-   ```terminal
-   export PRISMA_QUERY_ENGINE_BINARY=/custom/my-query-engine-unix
-   ```
+```terminal
+export PRISMA_QUERY_ENGINE_BINARY=/custom/my-query-engine-unix
+```
 
    </tab>
 
    <tab>
 
-   ```terminal
-   set PRISMA_QUERY_ENGINE_BINARY=c:\custom\my-query-engine-windows.exe
-   ```
+```terminal
+set PRISMA_QUERY_ENGINE_BINARY=c:\custom\my-query-engine-windows.exe
+```
 
    </tab>
 
 </TabbedContent>
 
+#### Navigating a corperate firewall with <inlinecode>PRISMA_BINARIES_MIRROR</inlinecode>
```

Comment:
This is only one of many use cases. This is much more guide-y than the concept documentation usually is.

Fundamentally you just
> ## Define a download mirror with `PRISMA_BINARIES_MIRROR`

Reply:
Hmm. Do we want to give any examples in the ceoncepts section then? As im learning what this does, this is the use case that came up when we were speaking so thats the one i used. I feel like an explanation is needed as to what `Define a download mirror with PRISMA_BINARIES_MIRROR` means, hence the example.

If i was coming to this page for the first time i wouldnt have a clue what that meant.

Reply:
Then the explanation is that you can set a different base URL for the downloading of the binaries, that is used instead of our default URL (binaries.prisma.sh). That is the concept as I understand it.

"Navigating a corporate firewall" then might be one of the use cases or problems that can be solved with it.